diff --git a/config.py b/config.py
index d321073..2588691 100644
--- a/config.py
+++ b/config.py
@@ -1,5 +1,5 @@
 config = {"epochs": 100, 
-          "batch_size": 128, 
-          'lr':0.001,
+          "batch_size": 16, 
+          'lr':0.01,
           'test': True, #if set to true, the training set size would be 3000 for testing 
           'train_date': '0709'}
\ No newline at end of file
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index c9f5824..9cc19dc 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20210709_035732-uap4byg0/logs/debug-internal.log
\ No newline at end of file
+run-20210709_043829-ow6xmj32/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index e9ff34e..e310770 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20210709_035732-uap4byg0/logs/debug.log
\ No newline at end of file
+run-20210709_043829-ow6xmj32/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index f13f529..30a2214 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20210709_035732-uap4byg0
\ No newline at end of file
+run-20210709_043829-ow6xmj32
\ No newline at end of file
